content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -151,6 +151,13 @@ and also set a value for the `ImageMaximumGCAge` field in the kubelet configurat
 The value is specified as a Kubernetes _duration_; for example, you can set the configuration
 field to `3d12h`, which means 3 days and 12 hours.
 
+{{< note >}}
+This feature does not track image usage across Kubelet reboots. If the Kubelet
+is rebooted, the image age is reset, causing the Kubelet to wait the full
```

Comment:
The kubelet doesn't have a bootloader, so:
```suggestion
```suggestion
This feature does not track image usage across kubelet restarts. If the kubelet
is restarted, the tracked image age is reset, causing the kubelet to wait the full

```